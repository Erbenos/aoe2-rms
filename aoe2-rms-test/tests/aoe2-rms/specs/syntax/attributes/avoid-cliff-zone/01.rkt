#lang aoe2-rms
; Fill the map with stone that stays 3 tiles away from cliffs.

<CLIFF-GENERATION>
<OBJECTS-GENERATION>
(create-object 'STONE
               (number-of-objects 9999)
               (avoid-cliff-zone 4))
