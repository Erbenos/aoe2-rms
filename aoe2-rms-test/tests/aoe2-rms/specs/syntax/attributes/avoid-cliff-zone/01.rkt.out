<CLIFF_GENERATION>
<OBJECTS_GENERATION>
create_object STONE {
  number_of_objects 9999
  avoid_cliff_zone 4
}
